### ISSUE TITLE
Bug: Press Release whose posted date (year) changed is still showing in the previous year

### ISSUE THREAD

**Issue by VictoriaSunNIH** [NCIOCPL/cgov-digital-platform#1229]
Steps to replicate the issue:

1. Edit an existing press release (I picked AI approach outperformed human experts in identifying cervical precancer)
2. Change all 3 dates to a different year (I changed all three dates to be in 2018).
3. Go view the yearly archive pages for Press Release, and note that while the date displayed for this PR is now changed to be 2018, it's now appearing on both the 2019 listing, and the 2018 listing.  It should only appear on the 2018 listing, and no longer on the 2019 listing.

![image](https://user-images.githubusercontent.com/31222859/55024151-a87df980-4fd4-11e9-8a0a-c11ec035e6f4.png)

![image](https://user-images.githubusercontent.com/31222859/55024165-b0d63480-4fd4-11e9-98ea-f92d62386de1.png)

**Comment by suryatheja-muppalla** [comment]
Approach:

Unable to reproduce the issue as the year changes are reflecting rightly in the Press Archive page.
No code change or configuration change is required.
Content editors shall make sure to follow below steps to update the year for Press nodes:
Step 1: Navigate to Edit page of any Press node
Step 2: Update ‘Posted Date’ field value with Year
Step 3: ‘Change to’ field value could be ‘Archived’ or ‘Editing’
Step 4: Save the changes -> At this point changes are not yet published
Step 5: Edit the same node and now after reviewing the data, make sure to select ‘Publish’ for ‘Change to’ field value.
Step 6: Save the node -> Now changes are published and updated Year will reflect in the Press filter page.

Cc: @VictoriaSunNIH  @bryanpizzillo

**Comment by bryanpizzillo** [comment]
This needs to be tested on an ACSF environment. (Somewhere that used Akamai)

**Comment by alinai** [comment]
@suryatheja-muppalla , I could not reproduce the issue on test-acsf. 

I executed the following steps to reproduce:
1. Edited an existing press release from the Year- 2021 (I picked the press release "Fecal microbiota transplants help patients with advanced melanoma respond to immunotherapy")
2. Changed all 3 dates to a different year (2020) (I changed all three dates to be in 2020).
3. Viewed the yearly archive pages for Press Release, and noticed that the date displayed for this Press Release is now changed to be 2020 and it is now appearing only on the 2020 listing and no longer appearing on the 2021 listing.

Note that when we view the Press Prelease with changed date the breadcrumb still shows that it appears under 2021 which is confusing.

![image](https://user-images.githubusercontent.com/28455973/131742452-b3163377-68c6-45f6-9bf5-e14e02170087.png)
